src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -35,33 +36,65 @@ const BalanceRow = styled.div`
   align-items: center;
   display: flex;
   flex-direction: row;
-  font-size: 20px;
-  justify-content: space-between;
-  line-height: 28px;
-  margin-top: 12px;
+  margin-top: 20px;
 `
 const BalanceItem = styled.div`
   display: flex;
   align-items: center;
 `
 
+const BalanceContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-left: 8px;
+  flex: 1;
+`
+
+const BalanceAmountsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const StyledNetworkLabel = styled.div`
+  color: ${({ theme }) => theme.accentAction};
```

Comment:
Could we use colors from chainInfo.ts here? to follow the design's difference in color between chains. see below

<img width="113" alt="image" src="https://user-images.githubusercontent.com/39385577/210664085-e1920ddb-0c7e-4235-ad85-455289168ef9.png">